docs/docs/debugging-async-lifecycles.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: Debugging Asynchronous Lifecycle Methods
+title: Depuración de métodos de ciclo de vida asíncrono
 ---
 
-Various lifecycle methods (see: [Gatsby Node APIs](/docs/node-apis/)) within Gatsby are presumed to be asynchronous. In other words, these methods can _eventually_ resolve to a value and this value is a [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise). We wait for the `Promise` to resolve, and then mark the lifecycle method as completed when it does.
+Se presume que varios métodos de ciclo de vida (consulta: [Gatsby Node APIs](/docs/node-apis/)) dentro de Gatsby son asíncronos. En otras palabras, estos métodos pueden _eventualmente_ resolverse en un valor y este valor es una [`Promesa`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise). Esperamos a que se resuelva la `Promesa`, y luego marcamos el método de ciclo de vida como completado cuando lo hace.
```

Comment:
As there's a Spanish translation of that page, I'd replace the English version with it.
Spanish version: https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Objetos_globales/Promise.